### DISCUSSION TITLE
zsh: command not found: now

### DISCUSSION THREAD

**Discussion by kodcuserkan** [vercel/vercel#3975] [General] [answered]
I downloaded now by typing "npm i -g now" on my macOS ,
Then after downloaded typed 
`now` 
but I got 
`zsh: command not found: now`
I reinstalled node js , I restarted everything but no "now" command not works for me. 
My os version : macOs catalina 10.15.4 (19E266)
Thanks in advance

**Comment by paulogdm** [comment, marked as answer]
This is unfortunately not related to our CLI. Please check:

- https://stackoverflow.com/questions/37189081/react-native-command-not-found
- https://stackoverflow.com/questions/12743928/command-not-found-after-npm-install-in-zsh
- https://stackoverflow.com/a/15692703

**Reply by kodcuserkan** [reply]
thanks man

**Reply by Naweeda** [reply]
I am having the same issue, @kodcuserkan how were you able to solve it? Can you please share?
